ringtail/storagemanager.py

```diff
@@ -182,125 +173,54 @@ def insert_data(
         self,
         results_array,
         ligands_array,
-        interaction_array,
+        interaction_list,
```

Comment:
Not a priority for this PR I think, but it might be a good idea to see if we can do more type hinting as we continue to iterate on this code.